frontend/src/api/k8s/servingRuntimes.ts

```diff
@@ -114,25 +114,25 @@ export const assembleServingRuntime = (
     updatedServingRuntime.spec.containers[0]?.resources,
   );
 
-  updatedServingRuntime.spec.containers = servingRuntime.spec.containers.map(
-    (container): ServingContainer => {
-      const volumeMounts = container.volumeMounts || [];
-      if (!volumeMounts.find((volumeMount) => volumeMount.mountPath === '/dev/shm')) {
-        volumeMounts.push(getshmVolumeMount());
-      }
+  updatedServingRuntime.spec.containers = servingRuntime.spec.containers.map(function mapContainer(
+    container,
+  ): ServingContainer {
+    const volumeMounts = container.volumeMounts || [];
+    if (!volumeMounts.find((volumeMount) => volumeMount.mountPath === '/dev/shm')) {
+      volumeMounts.push(getshmVolumeMount());
+    }
 
-      return {
-        ...container,
-        resources: isModelMesh ? resources : resourceSettings,
-        affinity,
-        volumeMounts,
-      };
-    },
-  );
+    return {
+      ...container,
+      resources: isModelMesh ? resources : resourceSettings,
+      affinity,
+      volumeMounts,
+    };
+  });
 
-  if (isModelMesh) {
-    servingRuntime.spec.tolerations = tolerations;
-  }
+  updatedServingRuntime.spec.tolerations = isModelMesh
+    ? tolerations
+    : servingRuntime.spec.tolerations;
```

Comment:
Why this change?
Wouldn't it just be:
```
  if (isModelMesh) {
    updatedServingRuntime.spec.tolerations = tolerations;
  }
```

Reply:
@christianvogt Yes, this can also be done. I am trying to understand, would the new syntax can cause the issue?

Reply:
Looking at the old code, it didn't do an assignment if `!isModelMesh` so I'm trying to keep the logic as close to the same when making these lint changes.
Is the else case you added not simply assigning the exact same value that is already present since `updatedServingRuntime` starts off as `servingRuntime` ? If so, this assignment is unnecessary.